espnet2/train/trainer.py

```diff
@@ -83,7 +118,7 @@ def add_arguments(cls, parser: argparse.ArgumentParser):
     @classmethod
     def train_one_epoch(
         cls,
-        model: AbsE2E,
+        model: torch.nn.Module,
```

Comment:
I loosen the type check for AbsE2E because it can be also DistributedDataParallel.